### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#307] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181328](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/e5ba0667-94d3-4ea6-8018-61f486373cd7)
![Screenshot 2023-11-04 181342](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8c40b016-6858-4251-88b5-1b053ca4aade)

**Comment by IonutCicio** [comment]
#### 4.1

Scrivere tutti gli elementi di $S_3$

$$
S_3 = \Biggl\\{
\begin{pmatrix} 1 & 2 & 3\\
1 & 2 & 3 \end{pmatrix}, 
\begin{pmatrix} 1 & 2 & 3\\
1 & 3 & 2 \end{pmatrix},
\begin{pmatrix} 1 & 2 & 3\\
2 & 1 & 3 \end{pmatrix},
\begin{pmatrix} 1 & 2 & 3\\
2 & 3 & 1 \end{pmatrix},
\begin{pmatrix} 1 & 2 & 3\\
3 & 1 & 2 \end{pmatrix},
\begin{pmatrix} 1 & 2 & 3\\
3 & 2 & 1 \end{pmatrix}
\Biggl\\}
$$

#### 4.2

Ho usato una notazione per cui $213$ indica che $\tau(1) = 2, \tau(2) = 1, \tau(3) = 3$

| $\circ$ | 123 | 132 | 213 | 231 | 312 | 321 |
|--|--|--|--|--|--|--|
| **123** | 123 | 132 | 213 | 231 | 312 | 321 |
| **132** | 132 | 123 | 231 | 213 | 321 | 312 |
| **213** | 213 | 312 | 123 | 321 | 132 | 231 |
| **231** | 231 | 321 | 132 | 312 | 123 | 213 |
| **312** | 312 | 213 | 321 | 123 | 231 | 132 |
| **321** | 321 | 231 | 312 | 132 | 213 | 123 |

#### 4.3

Fra i possibili ordini c'è $1$, dato che molti elementi, composti con se stessi, danno l'identità $123$, basta analizzare la diagonale.

#### 4.4

| $\tau$ | $o(\tau)$ | Spiegazione |
|:--:|:--:|:--|
| 123 | 1 | $123 \circ 123 = 123$ |
| 132 | 1 | $132 \circ 132 = 123$ |
| 213 | 1 | $213 \circ 213 = 123$ |
| 231 | 2 | $231 \circ 231 = 312; 321 \circ 231 = 123$ |
| 312 | 2 | $312 \circ 312 = 231; 231 \circ 312 = 123$ |
| 321 | 1 | $321 \circ 321 = 123$ |

**Comment by CiottoloMaggico** [comment]
Utilizzo stessa notazione di @CuriousCI per esprimere la tabella 
Tabella moltiplicativa mostra come S3 non sia un gruppo commutativo
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/18a5f3a6-cc19-42cd-bc26-2153987d201d)

**Comment by Elia-Belli** [comment]
- **(4.1)** Tutte gli elementi di $S_3$ sono tutte le possibili bigezioni da $\\{1,2,3\\}$ a $\\{1,2,3\\}$, che sono in numero $3!=6$, elencate da  @CuriousCI 
- **(4.3)** Come dice @CiottoloMaggico i possibili ordini degli elementi in $S_3$ sono $1,2,3$: @CuriousCI forse intendevi che hanno quasi tutti ordine $2$ infatti gli elementi sulla diagonale sono tutti del tipo $xxx\circ xxx=xxx^2$
- **(4.7)** Per verificare che tale sottogruppo è ciclico di ordine $2$ basta comporre $2$ volte il generatore con se stesso e osservare che si ottengono tutti gli elementi $H$.
Inoltre come mostra @CiottoloMaggico $aH\neq Ha$ infatti la composizione non gode di commutatività in $S_3$
